Cargo.toml

```diff
@@ -1,23 +1,21 @@
 [package]
 name = "crc"
-version = "1.8.1"
+version = "1.9.0"
```

Comment:
no need for major version bump for feature modification?

Reply:
This could be a breaking change. If it's versioned as 1.8.2, then semantic versioning considers it a patch and it could break a lot of builds. Maybe you want to couple this with other changes for the 1.9.0 release?

I actually think this might even warrant a 2.0.0, since it can be considered an "incompatible API change" but I'm not sure.

Reply:
if we're looking at major version bump, please allow some time for us to consolidate interfaces, e.g. #45 and other improvements we may think of

@zachmse any thoughts here?

Reply:
@mrhooray  @akhilles
I think this would require a 2.0.0 release, and as such, I would agree that we should hold off until we figure out all the other potentially breaking changes as well. Perhaps we should create a tracking issue to keep track of the various 2.0.0 changes?